### DISCUSSION TITLE
Instructions how to update AP snakemake

### DISCUSSION THREAD

**Discussion by jkosinski** [KosinskiLab/AlphaPulldown#561] [Q&A]
Could you add, perhaps here:
https://github.com/KosinskiLab/AlphaPulldown/blob/e35bf48f96308b02464738a777482ee2226c7032/README.md?plain=1#L269
what to do if a user has:
1. deployed the pipeline with snakedeploy deploy-workflow
2. edited config.yaml and added the list files
3. run the pipeline and sth did not work (eg. bug or whatever
and then we fixed the AP and then the user wants to
4. update the AP snakemake and re-run in the same folder and reusing the previous config.yaml

From the provided instruction, I would just re-run snakedeploy deploy-workflow but would they work and not delete my config and list files?

(I am exactly in this situation right now ;-)

**Comment by DimaMolod** [comment]
I think if you use `--branch main`, you just re-run without even re-deploying. And if you use some release tag, you'll need to re-deploy to the new folder using the new tag (given there is one after we fixed a bug) and then copy the config.yaml file into the new config folder. Is that correct, @maurerv?

**Comment by maurerv** [comment]
Things should be stable before you deploy :)

Running snakedeploy will always remove and fetch all files that are included in the particular tag. So your changes to config.yaml and sample_sheet.csv will be overwritten

**Comment by jkosinski** [comment]
From what I see in our lab and from github issues, we constantly update AP and re-run, even in between having already some models calculate, and I wouldn't know from the current doc what is the correct procedure for re-running after update. Could it be somehow explained (could be just a link from the main readme to the wiki).
